server/app/controllers/DisplayableMessage.java

```diff
@@ -0,0 +1,39 @@
+package controllers;
+
+import views.components.ToastMessage.ToastType;
+
+public class DisplayableMessage {
```

Comment:
Please mark this class as `final` to make the decision to allow subclassing explicit rather than implicit.